### ISSUE TITLE
Consistency for `tab -` behavior

### ISSUE THREAD

**Issue by TheTeXnician** [austinjones/tab-rs#167]
When running `tab -` (think of forgetting the`l` after it :wink:) I get the following output

```
error: Found argument '-/' which wasn't expected, or isn't valid in this context

USAGE:
    tab [FLAGS] [OPTIONS] [--] [TAB]

For more information try --help
```

which I would read as an illegal action resulting in a failure. Much to my surprise, however, a tab named `-` is generated and switched to. `tab -l` then shows

```
Available tabs:
    -/
```

Would it be possible to either

* disable the error if `-` is a valid tab name,
* turn the error into an informational message that a tab named `-` was created and that maybe this wasn't what the user intended to do or
* forbid the `-` tab name?

Many thanks for providing this nice tool.

**Comment by austinjones** [comment]
Ah, good catch.  Clap is parsing `tab -` as a positional argument, and `tab -/` as a flag with the short name `/`.

I think forbidding tab names that start with a hyphen is a good idea, as even if it got past this step, the name could be shadowed by CLI flags.

I'm glad you like the tool!  And thanks for the feature request :)

**Comment by austinjones** [comment]
I've implemented this in #170.  I'll update & close the issue once 0.4.0 is released.

**Comment by austinjones** [comment]
Closing, [v0.4.0](https://github.com/austinjones/tab-rs/releases/tag/v0.4.0) is released!
